lib/routes/chnmuseum/xingnew.ts

```diff
@@ -0,0 +1,74 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/zx/xingnew',
+    categories: ['other'],
+    example: '/zx/xingnew',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['chnmuseum.cn/zx/xingnew'],
+            target: '/zx/xingnew',
+        },
+    ],
+    name: '资讯要闻',
+    maintainers: ['ShabbyWhineYear'],
+    handler: async () => {
+
+        const response = await ofetch(`https://www.chnmuseum.cn/zx/xingnew/`);
```

Comment:
```suggestion
        const response = await ofetch('https://www.chnmuseum.cn/zx/xingnew/');
```
Only use template literals when necessary.